src/functions/module_lookup.ts

```diff
@@ -262,11 +237,11 @@ export const MATCH: AddFunctionDescription = {
 
 export const ROW: AddFunctionDescription = {
   description: _lt("Row number of a specified cell."),
-  args: args(
-    `cell_reference (meta, default=${_lt(
-      "The cell in which the formula is entered by default"
-    )}) ${_lt("The cell whose row number will be returned.")}`
-  ),
+  args: [arg(
+    `cell_reference (meta)`,
+    _lt("The cell whose row number will be returned."),
+    { default: _lt("The cell in which the formula is entered by default")}
```

Comment:
@LucasLefevre What do you think of this API ? I do not see another good solution to translate default value

Reply:
Ah yes, but we saw this particular default last week (I think you were working from home). This default value is actually not used like it should. It should be a real value, not a description for the value.
I think I would just remove it or move it to the arg description

Reply:
What about `true` and `false` ? It's a value, but should be translated IMO. But it can be always moved in the arg description, it's fine for me

Reply:
@laa-odoo had a similar thought this morning about argument names that are not translated (`cell_reference`).
Since we decided to not translate function names, I think it's acceptable to do the same for arg names and boolean values.
Description are another story since they are real English sentences.